### PR TITLE
Fixed an issue where dropdown menus were ill-alligned

### PR DESCRIPTION
**Pull request by engelhamer**

### CODE REVIEW

File: src/vue/src/components/assets/ThemeSelect.vue [hunk 1]
```diff
@@ -5,39 +5,41 @@
...
         @remove="(removedOption, id) => $emit('remove', removedOption, id)"
-        @open="() => { isOpen = true }"
-        @close="() => { isOpen = false }"
     >
-        <span slot="limit">
-            {{ (value && value.length) ? value.length : 'No' }} {{ multiSelectText }}
-        </span>
+        <template slot="tag">
+            {{ '' }}
+        </template>
```
Comment (Larspolo) [inline src/vue/src/components/assets/ThemeSelect.vue:25]: [NIT] Why is this necessary? Would sth like `<template slot="tag"/>` also work?

**Review by Larspolo — APPROVED** [review]

File: src/vue/src/components/assets/ThemeSelect.vue (hunk 1 above)
Comment (mjvkeulen) [reply src/vue/src/components/assets/ThemeSelect.vue:25]: Don't forget the shorthand `#tag`

File: src/vue/src/components/entry/Entry.vue [hunk 2]
```diff
@@ -268,7 +268,8 @@ export default {
         safeToLeave () {
             // It is safe to leave the entry if it is not currently being edited AND if no content for an entry to be
             // created is provided.
-            return !this.edit && !(this.create && this.template.field_set.some(field => this.newEntryContent[field.id]))
+            return !this.edit && !(
+                this.create && this.template.field_set.some((field) => this.newEntryContent[field.id]))
```
Comment (mjvkeulen) [inline src/vue/src/components/entry/Entry.vue:272]: [NIT]: This is introduced via linter fix for the function parenthesis, would same line not still be allowed?

**Review by mjvkeulen — APPROVED** [review]
